manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Resolume.Arena
+PackageVersion: 7.15.0.23381
+PackageLocale: en-US
+Publisher: Resolume B.V.
+PublisherUrl: https://resolume.com
+PublisherSupportUrl: https://resolume.com/support/
+PrivacyUrl: https://resolume.com/company/privacy
+Author: Resolume B.V.
+PackageName: Resolume Arena
+PackageUrl: https://resolume.com/download
+License: Proprietary
+Copyright: Copyright ©2001-2023 Resolume B.V. All rights reserved
+ShortDescription: Resolume Arena
```

Comment:
```suggestion
ShortDescription: Similar to Resolume Avenue, but with advanced options for projection mapping and blending
```

Reply:
Updated the description and short description